### ISSUE TITLE
Google Pay Standalone integration 

### ISSUE THREAD

**Issue by HareshIndiaNIC** [Shopify/mobile-buy-sdk-android#662]
**Google Pay Standalone integration **

I want to add a standalone button for Google play. Shopify support like this functionality without web check out. 

Shopify removes android pay support so Google Pay supports standalone as SDK.

Also notice in the sample code of Shopify SDK comment Google Pay related code in a comment. 

Like in Gradle file ANDROID_PAY_PUBLIC_KEY is unused.

**Comment by kapilArsr** [comment]
Yes i am facing same problem Google pay code is commented it out if anyone have solution please help me out
